### ISSUE TITLE
Bitdefender, Threat detected MalwareByte the same.

### ISSUE THREAD

**Issue by Manniala** [zanllp/sd-webui-infinite-image-browsing#750]
Hey, every time i start up the "Infinite Image Browsing.exe" i get a warning.
![image](https://github.com/user-attachments/assets/2ca59415-4043-40ac-8652-42ffae1abcb4)

And when i scan the iib_api_server.exe i get this.
![image](https://github.com/user-attachments/assets/8714e1e3-c6f7-40ab-9086-c823498a105c)

I know it is most likely a false posictive, but when MalwareByte also reports something.
![image](https://github.com/user-attachments/assets/e9a648a1-eacd-4728-87d3-5db5ec7c4f18)

I thought i would mentikon this, installed from "Infinite.Image.Browsing_1.2.0_x64-setup.exe", scanning the installer it self with Bitdefender Total Security showed nothing, but when i scanned it with MalwareByte i did get this:
File: 1
Trojan.Dropper.NSIS, C:\USERS\DUDES\DOWNLOADS\PROGRAMS\INFINITE.IMAGE.BROWSING_1.2.0_X64-SETUP.EXE, No Action By User, 5190, 1283878, 1.0.91632, , ame, , 36DD809440D3D914856B3A4E3C2C45F7, CBD342850C981D2621AAE364587C0656CC6B5041DB75BB660A6916FF607A34CB

I than tried to download and ran "Infinite.Image.Browsing_1.2.0_x64-setup-pyinstaller.exe", this one starts up without any problem, no warning, and when i look in the folder "iib_api_server.exe" got a icon, as i recalled the other one did not, and i can execute this one, the other one resulted in Bitdefender allerting me of malware.

**Comment by Manniala** [comment]
Ah and i tried to download "Source code (zip) Sep 28", it is also under the section 1.2,0, but it say its v1.1.0 after running it.

Version: v1.1.0 (standalone)
Hash: 72147d6cdb51dede5011f398baa5fb12392c3beb
Not the latest commit
Latest Commit: 7215a4cadfc14151a3ef8e036ecb0ba8e27d8a68 (Updated at 2024-11-10T10:28:55Z)

And you get this in the menu, telling there is a new version, but the nevest version that i can get is the 1.1.0
![image](https://github.com/user-attachments/assets/e806ed04-1129-4b14-b370-93783009283a)

**Comment by zanllp** [comment]
The current situation cannot fully eliminate these false positives, which is an unavoidable issue when using Python as a sidecar.

**Comment by zanllp** [comment]
> Ah and i tried to download "Source code (zip) Sep 28", it is also under the section 1.2,0, but it say its v1.1.0 after running it. Running it as stand alone, for web
> 
> Version: v1.1.0 (standalone) Hash: [72147d6](https://github.com/zanllp/sd-webui-infinite-image-browsing/commit/72147d6cdb51dede5011f398baa5fb12392c3beb) Not the latest commit Latest Commit: [7215a4c](https://github.com/zanllp/sd-webui-infinite-image-browsing/commit/7215a4cadfc14151a3ef8e036ecb0ba8e27d8a68) (Updated at 2024-11-10T10:28:55Z)
> 
> And you get this in the menu, telling there is a new version, but the nevest version that i can get is the 1.1.0 ![image](https://private-user-images.githubusercontent.com/32381860/385181036-e806ed04-1129-4b14-b370-93783009283a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE0MTQwNTEsIm5iZiI6MTczMTQxMzc1MSwicGF0aCI6Ii8zMjM4MTg2MC8zODUxODEwMzYtZTgwNmVkMDQtMTEyOS00YjE0LWIzNzAtOTM3ODMwMDkyODNhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTEyVDEyMTU1MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMxN2QxMTBhZTU2NTAyYTQ3MzUyOWJjZjM0ZDJmMTRkMzJmMTA0ZGQ1ZTE2NjgxZWNmYmI2NTM3NGYzNDhmNDcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.NnALYuBj6UqagafxD4yR8TkZuXxoHlFS9rs_gtY2zPA)

This issue seems to be because I released it twice, but the source code zip file was not updated. I recommend using the code from the main branch directly.

**Comment by Manniala** [comment]
Yeah, i would also assume it was a false positive, or many more would had been writing about it.
I have submittet this to [Bitdefender ](https://www.bitdefender.co.uk/consumer/support/answer/88563/) via their homepage, i suggest you do the same.

It is easy, and takes a minute or so, they have actually fixed another problem i had some while back with SD Prompt Reader, the creator and i reported it, and a little while later, it no longer flagged it.

If that were just cuinsident i do not know, or if they really takes this serious, well not sure, but i think it is in Bitdefenders own interest to eliminate as many false positive as they can.
